src/annotator/anchoring/xpath-util.js

```diff
@@ -50,58 +53,35 @@ function getNodePosition(node) {
   return pos;
 }
 
-/**
- * A simple XPath evaluator using jQuery which can evaluate queries of
- *
- * @deprecated
- */
-export function simpleXPathJQuery(nodes, relativeRoot) {
-  const paths = nodes.map((index, node) => {
-    let path = '';
-    let elem = node;
-    while (elem.nodeType === Node.ELEMENT_NODE && elem !== relativeRoot) {
-      let tagName = elem.tagName.replace(':', '\\:');
-      let idx = $(elem.parentNode).children(tagName).index(elem) + 1;
-      path = '/' + elem.tagName.toLowerCase() + `[${idx}]` + path;
-      elem = elem.parentNode;
-    }
-    return path;
-  });
-  return paths.get();
-}
-
 function getPathSegment(node) {
   const name = getNodeName(node);
   const pos = getNodePosition(node);
   return `${name}[${pos}]`;
 }
 
 /**
- * A simple XPath evaluator using only standard DOM methods which can
- * evaluate queries of the form /tag[index]/tag[index].
+ * A simple XPath generator using which can generate XPaths of the form
```

Comment:
Something is quit right with this sentence.. previously it was 

```
 * A simple XPath evaluator using only standard DOM methods which can
 * evaluate queries of the form /tag[index]/tag[index].

```

Perhaps

`* A simple XPath evaluator which can evaluate queries of the form /tag[index]/tag[index].`

Reply:
The updated comment here is correct - this function generates rather than evaluates XPaths. There is indeed a grammatical error ("generator using which can generate") which I'll correct.